wolfssl/wolfcrypt/settings.h

```diff
@@ -409,6 +412,16 @@
     #endif
 #endif
 
+#ifdef WOLFSSL_EMBOS
+    #define USE_FAST_MATH
+    #define HAVE_ECC
+    #define NO_DEV_RANDOM
```

Comment:
We used the #warning for testing purposes only generate seed for the sake of time. We can research this more.